src/app/AdminUser.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App;
+
+use Illuminate\Contracts\Auth\MustVerifyEmail;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+use Illuminate\Notifications\Notifiable;
+
+class AdminUser extends Authenticatable
+{
+    use Notifiable;
+
+    protected $guard = 'admin';
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name', 'email', 'password',
```

Comment:
[imo]
改行してあげるほうがより綺麗かと個人的に思いました。
```suggestion
        'name',
        'email',
        'password',
```

Reply:
綺麗で見やすいです！そのようにしてみます。ありがとうございます。